app/src/components/inputs/autocompletes/singleAutocomplete/singleAutocomplete.scss

```diff
@@ -97,3 +97,7 @@
     }
   }
 }
+.border-view {
```

Comment:
You are removing the `border` for border view? It looks contradictory.
May be better to call the property and the class f.e. `nakedView` or smth like that.